tests/PySys/tedge_mapper_az/tedge_mapper_az_without_timestamp/run.py

```diff
@@ -0,0 +1,89 @@
+from pysys.basetest import BaseTest
+
+import time
+import json
+
+"""
+Validate a tedge-mapper-az message without timestamp that is published
+on az/messages/events/
+
+Given a configured system
+When we set az.mapper.timestamp false
+When we start the tedge-mapper-az systemctl service
+When we start tedge sub with sudo in the background
+When we publish a Thin Edge JSON message
+Then we kill tedge sub with sudo as it is running with a different user account
+Then we validate the JSON message in the output of tedge sub
+Then we stop the tedge-mapper-az systemctl service
+Then we unset az.mapper.timestamp
```

Comment:
Oh, that is new to me. Do we maybe have a list of configuration options somewhere?

Reply:
The output of `tedge config list --doc` is pretty much that you wanted.
`az.mapper.timestamp` is documented in https://github.com/thin-edge/thin-edge.io/blob/main/docs/src/architecture/mapper.md#configure-whether-adding-a-timestamp-or-not.